src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -0,0 +1,275 @@
+href|string|[RFC 3986](https://tools.ietf.org/html/rfc3986)|Hyperlink to the resource
+
+### <a name="schema-listofrequest"></a>List of Request
+
+Name|Type|Format|Description
+---|---|---|---
+data|array|[Request](#schema-request)|List of Requests in the page requested
+operations|array|[Link](#schema-link)|Links to next, prev, first and last pages
+
+### <a name="schema-location"></a>Location
+
+Name|Type|Format|Description
+---|---|---|---
+countryCode|string|[ISO 3166-1](https://www.iso.org/iso-3166-country-codes.html)|The ISO 3166-1 country code
+countrySubDivisionCode|string|[ISO 3166-2](https://www.iso.org/standard/63546.html)|The ISO 3166-2 country sub code
+city|string|-|The city name of the location
+iataCode|string|-|The IATA code of the location
+name|string|-|The name of the location
+
+### <a name="schema-resourcelink"></a>ResourceLink
+
+Name|Type|Format|Description
+---|---|---|---
+href|string|[RFC 3986](https://tools.ietf.org/html/rfc3986)|Hyperlink to the resource
+id|string|[RFC 4122](https://tools.ietf.org/html/rfc4122)|Unique identifier of the related object
+template|string|-|Hyperlink template to the resource
+
+### <a name="schema-requestlink"></a>RequestLink
+
+Name|Type|Format|Description
+---|---|---|---
+requestId|string|4 to 6 alphanumeric characters|The public key of the Request (unique per customer)
+
+### <a name="schema-requesttype"></a>Request Type
+
+Name|Type|Format|Description
+---|---|---|---
+code|string|-|The code of the type inherited from the Request Policy type
+label|string|-|The label of the type inherited from the Request Policy Type
+
+### <a name="schema-expected-expense"></a>Expected Expense
+
+Name|Type|Format|Description
+---|---|---|---
+allocations|object|[Allocation](#schema-allocation)|The details of the allocations for this expected expense.
+approvedAmount|object|[Amount](#schema-amount)|The approved amount of the expected expense entry, in the transaction currency of the Request
+budgetAccrualDate|timestamp|[RFC 3339](https://tools.ietf.org/html/rfc3339)|The date to determine which budgets are affected (in the format yyyy-MM-dd'T'HH:mm:ss.SSS'Z')
+businessPurpose|string|-|The business purpose of the Request entry
+comments|object|[ResourceLink](#schema-resourcelink)|The list of comments for this expected expense
+custom1 to custom40|object|[CustomField](#schema-customfield)|The details from the Custom fields. <br>These fields may not have data, depending on the configuration
+exchangeRate|object|[Exchange Rate](#schema-exchangerate)|The exchange rate that applies to the entry
+expenseType|object|[Expense Type](#schema-expensetype)|The expense type of the entry. <br>**Required** for expected expenses, automatically set for segments depending on the SegmentType code.
```

Comment:
Where do they get the expense type? I don't see any calls that provide those values.

Reply:
This is indeed an issue. Currently we don't expose any endpoint  allowing to discover the expense types. The only way to know the data of an expense type is to create once one manually and get its details from the API. Here we are missing Expense v4 API to help us.